crates/manifest/src/lib.rs

```diff
@@ -205,6 +205,7 @@ impl From<ApplicationTrigger> for ApplicationTriggerSerialised {
 
 /// HTTP trigger configuration.
 #[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
+#[serde(default)]
```

Comment:
what does this change do?

Reply:
It uses `Self::default` to set default values for any missing fields.